docs/content/faq.md

```diff
@@ -250,23 +249,16 @@ See [#1247][github-issue-1247] for an example.
 
 ### Common Errors
 
+Due to the way we set `mydestination`, you might encounter the following issue:
```

Comment:
```suggestion
Normally you will assign DMS a `hostname` such as `mail.example.com`. If you instead use a bare domain (_such as `example.com`_) or add an alias / account with the same value as your `hostname`, this can cause a conflict due to mail addressed to `hostname` being configured for system account mail only via Postfix `main.cf` setting `mydestination`.

When this conflict is detected you'll find logs similar to this:
```